lib/body.js

```diff
@@ -21,36 +21,36 @@ class Body {
 
     // Get the skills parameters from the parameter server to extract the
     // possible motions.
-    const param = new Param({
+    const parameter = new Param({
```

Comment:
What is wrong with param? It's an instance of Param which is in the roslibjs api

Reply:
XO wants it updated

Reply:
Ok, fine 😆